great_expectations/util.py

```diff
@@ -405,12 +405,16 @@ def is_library_loadable(library_name: str) -> bool:
 def load_class(class_name: str, module_name: str):
     if class_name is None:
         raise TypeError("class_name must not be None")
+
     if not isinstance(class_name, str):
         raise TypeError("class_name must be a string")
+
```

Comment:
I would prefer if we didn't make these kinds of whitespace changes.
It muddies up the gitblame and makes it harder to follow when the file has substantively changed.

Reply:
@Kilo59 I am addressing your comments (including reverting the blank lines -- although I like them for readability, but I see your point) in the pull request that is coming up in 10 minutes -- thank you!